.github/workflows/publishing.yml

```diff
@@ -441,28 +497,28 @@ jobs:
           path: /tmp/wheels
           retention-days: 1
           if-no-files-found: error
+    
+  cudaq_wheels_release_info:  
+    name: Update release info of CUDA Quantum Python wheels
```

Comment:
Do you have a link to a run of this workflow? I am confused why this would work, because last I checked, outputs form matrix jobs required retrieving them from json (but maybe GitHub has finally added support for this?)